wordpress-mobile/WordPress-iOS
# Update Muriel colors to v2

Refs #11683

To test:
- ensure the app still builds
- observe that colors have updated (primary blue, red, and pink have all been changed, especially at the lighter shades)

Update release notes:
